src/corefx/System.Globalization.Native/configure.cmake

```diff
@@ -14,10 +14,18 @@ else()
     set(CMAKE_REQUIRED_LIBRARIES ${ICUCORE})
 endif()
 
+if(CMAKE_SYSTEM_NAME STREQUAL FreeBSD)
+    # Somehow check_symbol_exists() failed to link to the required icu libraries
+    # and hence failed the test and cause problem about using deprecated function
+    # later on. With icu58 (lastest from ports as of today), this can be forced
+    # and at least everything get built.
+    set(HAVE_SET_MAX_VARIABLE 1)
```

Comment:
Agree, this is meant to be temporary. Don't know much about cmake, tried a few things and couldn't get cmake to work. This is the last resort.